tests/unit/test_drivers.py

```diff
@@ -1,26 +1,87 @@
-from ni_measurementlink_service._drivers import closing_session
+import pytest
+
+from ni_measurementlink_service._drivers import closing_session_with_ts_code_module_support
 from tests.utilities import fake_driver
+from tests.utilities.fake_driver import SessionInitializationBehavior
 
 
-def test___closable_session___with_closing_session___session_closed() -> None:
-    with closing_session(fake_driver.ClosableSession("Dev1")) as session:
-        assert isinstance(session, fake_driver.ClosableSession)
+@pytest.mark.parametrize(
+    "initialization_behavior",
+    [
+        (SessionInitializationBehavior.AUTO),
+        (SessionInitializationBehavior.INITIALIZE_SERVER_SESSION),
+        (SessionInitializationBehavior.ATTACH_TO_SESSION_THEN_CLOSE),
+    ],
```

Comment:
These parentheses have no effect. Constructing a one-element tuple requires a comma, but you don't need a tuple for a single pytest parameter.

This applies to the other test functions too.

```suggestion
    "initialization_behavior",
    [
        SessionInitializationBehavior.AUTO,
        SessionInitializationBehavior.INITIALIZE_SERVER_SESSION,
        SessionInitializationBehavior.ATTACH_TO_SESSION_THEN_CLOSE,
    ],
```